src/main/java/nl/altindag/log/LogCaptor.java

```diff
@@ -147,6 +142,7 @@ private static LogEvent toLogEvent(ILoggingEvent iLoggingEvent) {
         String level = iLoggingEvent.getLevel().toString();
         String loggerName = iLoggingEvent.getLoggerName();
         ZonedDateTime timeStamp = ZonedDateTime.ofInstant(Instant.ofEpochMilli(iLoggingEvent.getTimeStamp()), ZoneOffset.UTC);
+        Map<String, String> mdcProperties = iLoggingEvent.getMDCPropertyMap();
```

Comment:
A small detail maybe... I prefer not to use abbreviations as it would be more difficult the understand. The getMDCPropertyMap just returns a `map` containing the `mapped diagnostic context`. What do you think of the variable name `diagnosticContext`? It is just a shorter version of `MDC` without the `Mapper`

Reply:
The method `iLoggingEvent.getMDCPropertyMap()` returns a mutable map. It is better to wrap it into a immutable map by using `Collections.unmodifiableMap()`

Reply:
Looks ok :)
Done x2

Reply:
Thank you very much!